src/main/java/core/basesyntax/strategy/impl/Return.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class Return implements OperationHandler {
+    @Override
+    public void handleFruitOperation(String fruit, Integer quantity) {
+        if (Storage.fruits.containsKey(fruit)) {
+            Storage.fruits.put(fruit, Storage.fruits.get(fruit) + quantity);
```

Comment:
Maybe create Dao for db would be better to use, against raw db usage

Reply:
I'm not sure if it is necessary in this case